src/control/cmd/ddb/commands_wrapper.go

```diff
@@ -91,9 +93,33 @@ func ddbClose(ctx *DdbContext) error {
 	return daosError(C.ddb_run_close(&ctx.ctx))
 }
 
-func ddbSuperblockDump(ctx *DdbContext) error {
+func ddbSuperblockDump(ctx *DdbContext) (*SuperBlock, error) {
+
 	/* Run the c code command */
-	return daosError(C.ddb_run_superblock_dump(&ctx.ctx))
+	cSb := C.struct_ddb_superblock{}
+	err := daosError(C.dv_superblock2(ctx.ctx.dc_poh, &cSb))
+	if err != nil {
+		return nil, err
+	}
+
+	/* Convert C uuit_t into a go UUID */
+	var goUuid uuid.UUID
+	cUUIDBytes := (*[16]byte)(unsafe.Pointer(&cSb.dsb_id))[:]
+	copy(goUuid[:], cUUIDBytes)
```

Comment:
This works, but there is also a `uuid.FromBytes` function that does some error checking as well. That might read a bit more clearly here.